server/config/buildTables.js

```diff
@@ -16,6 +16,15 @@ const userTableQuery = `
     "emailConfirmCode" VARCHAR(64),
     "createdOn" TIMESTAMPTZ DEFAULT now() NOT NULL
   );
+  CREATE TABLE IF NOT EXISTS books(
+    id SERIAL PRIMARY KEY,
+    title VARCHAR(100) UNIQUE NOT NULL,
+    body VARCHAR(100) NOT NULL,
+    description TEXT NOT NULL,
+    genre VARCHAR(100) NOT NULL,
+    pages NUMERIC(250) NOT NULL,
+    author VARCHAR(100) NOT NULL
```

Comment:
Sorry bro @chiboycalix , I think for better organization of data, author should be divested into its own table. The author column here in the books table should reference authors table if we are to go by the database model we designed.